### ISSUE TITLE
Issue templates are broken

### ISSUE THREAD

**Issue by dvberkel** [alliander-opensource/measure#13]
**Describe the bug**
When I create an issue, there are no templates to choice from.

**To Reproduce**
Open a new issue and you are not greeted with an option to choose from different templates

**Expected behavior**
I would expect that a choice between different templates.

**Comment by dvberkel** [comment]
The issue templates work again from commit 4abcfa7bf89a03e136e8f2a51410e622550399d1.

Unfortunately, this added the literal 

```
<!--
SPDX-FileCopyrightText: 2021-2022 Alliander N.V.

SPDX-License-Identifier: MPL-2.0
-->
```

**Comment by dvberkel** [comment]
Since the markdown file starts with a YAML block and the second document in this YAML file contains the template we are going to try to include the license information as a YAML comment

**Comment by dvberkel** [comment]
Fixed in commit 0723d30f8135a7d1b7247c7bc5d003aea03af36c
